hive/src/main/java/org/elasticsearch/hadoop/hive/EsStorageHandler.java

```diff
@@ -18,36 +18,39 @@
  */
 package org.elasticsearch.hadoop.hive;
 
-import java.util.Map;
-import java.util.Properties;
-
 import org.apache.commons.logging.Log;
 import org.apache.commons.logging.LogFactory;
 import org.apache.hadoop.conf.Configuration;
 import org.apache.hadoop.hive.metastore.HiveMetaHook;
 import org.apache.hadoop.hive.ql.metadata.DefaultStorageHandler;
+import org.apache.hadoop.hive.ql.metadata.HiveStoragePredicateHandler;
+import org.apache.hadoop.hive.ql.plan.ExprNodeDesc;
 import org.apache.hadoop.hive.ql.plan.TableDesc;
+import org.apache.hadoop.hive.serde2.Deserializer;
 import org.apache.hadoop.hive.serde2.SerDe;
 import org.apache.hadoop.mapred.InputFormat;
+import org.apache.hadoop.mapred.JobConf;
 import org.apache.hadoop.mapred.OutputFormat;
 import org.elasticsearch.hadoop.cfg.HadoopSettingsManager;
 import org.elasticsearch.hadoop.cfg.Settings;
+import org.elasticsearch.hadoop.hive.pushdown.Utils;
 import org.elasticsearch.hadoop.mr.EsOutputFormat;
 import org.elasticsearch.hadoop.mr.HadoopCfgUtils;
 import org.elasticsearch.hadoop.util.Assert;
 
-import static org.elasticsearch.hadoop.hive.HiveConstants.COLUMNS;
-import static org.elasticsearch.hadoop.hive.HiveConstants.COLUMNS_TYPES;
-import static org.elasticsearch.hadoop.hive.HiveConstants.TABLE_LOCATION;
+import java.util.Map;
+import java.util.Properties;
+
+import static org.elasticsearch.hadoop.hive.HiveConstants.*;
```

Comment:
Re-expand this import into the explicit types.

Reply:
done